### ISSUE TITLE
Firmware downloads?

### ISSUE THREAD

**Issue by Heclalava** [koolshare/koolcenter#28]
Hi

I used to download firmware from https://fw.koolcenter.com

However it's no longer loading. 
![Screenshot_20240807-173958_Brave](https://github.com/user-attachments/assets/3ddb5c45-6869-47d6-8c69-57ec15090151)

Is there an alternate download location? 

Thanks

**Comment by Heclalava** [comment]
Seems your site is blocking VPNs. I managed to get it to load after trying a few different VPNs. Why is that?

**Comment by maxwellcheng** [comment]
I got the same problem, any one can help? I just want to get the firmware for a old router...

**Comment by Heclalava** [comment]
> I got the same problem, any one can help? I just want to get the firmware for a old router...

Try turn off VPN or try VPN from different locations, then you can get in. I just checked now and I can visit the site using my current VPN location

**Comment by maxwellcheng** [comment]
Hey, thanks for the reply, Can you try:  
https://fw.koolcenter.com/Koolshare_RMerl_New_Gen_386/RT-AC68U/RT-AC68U_386.3_2_ca3f6fc_koolshare.trx 
See if you can download this, and may i know your VPN region if you can, thanks a lot.

**Comment by Heclalava** [comment]
> Hey, thanks for the reply, Can you try: https://fw.koolcenter.com/Koolshare_RMerl_New_Gen_386/RT-AC68U/RT-AC68U_386.3_2_ca3f6fc_koolshare.trx See if you can download this, and may i know your VPN region if you can, thanks a lot.

![Image](https://github.com/user-attachments/assets/8df3955c-c0db-48aa-867d-2a48324f6a64)

Nope I get a 404

**Comment by Heclalava** [comment]
> Hey, thanks for the reply, Can you try: https://fw.koolcenter.com/Koolshare_RMerl_New_Gen_386/RT-AC68U/RT-AC68U_386.3_2_ca3f6fc_koolshare.trx See if you can download this, and may i know your VPN region if you can, thanks a lot.

https://fw.koolcenter.com/KoolCenter_Merlin_New_Gen_386/RT-AC68U/RT-AC68U_386.3_2_ca3f6fc_koolshare.trx

Is this not what you are looking for? Your link seems broken.

**Comment by maxwellcheng** [comment]
Oh, I can this link from the link for 386.3_2: 
https://www.koolcenter.com/article/firmware/6_download/merlin_mod/70_rt-ac68u_merlin_mod/
![Image](https://github.com/user-attachments/assets/6e006572-0a36-46eb-99e1-dd840ef1ebe5)

Strange that you got 404, but I always got ERR_CONNECTION_TIMED_OUT...

why are they blocking my region...
many thanks for your help.

**Comment by Heclalava** [comment]
> Oh, I can this link from the link for 386.3_2: https://www.koolcenter.com/article/firmware/6_download/merlin_mod/70_rt-ac68u_merlin_mod/ ![Image](https://github.com/user-attachments/assets/6e006572-0a36-46eb-99e1-dd840ef1ebe5)
> 
> Strange that you got 404, but I always got ERR_CONNECTION_TIMED_OUT...
> 
> why are they blocking my region... many thanks for your help.

Yeah I am not sure why it is blocked in some regions. I get through with a VPN in Switzerland.
